src/css/components/navbar.css

```diff
@@ -4,17 +4,31 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+  flex-wrap: wrap;
+
+  @media (min-width: 759px) {
```

Comment:
Can you can use `@media (--hiq-upper-font-range) { ... }` which is HiQ's media query?